components/camel-properties/src/main/java/org/apache/camel/component/properties/PropertiesComponent.java

```diff
@@ -350,36 +346,12 @@ public void setIgnoreMissingLocation(boolean ignoreMissingLocation) {
 
     @ManagedAttribute(description = "Prefix token")
     public String getPrefixToken() {
-        return prefixToken;
-    }
-
-    /**
-     * Sets the value of the prefix token used to identify properties to replace.  Setting a value of
-     * {@code null} restores the default token (@link {@link #DEFAULT_PREFIX_TOKEN}).
-     */
-    public void setPrefixToken(String prefixToken) {
-        if (prefixToken == null) {
-            this.prefixToken = DEFAULT_PREFIX_TOKEN;
-        } else {
-            this.prefixToken = prefixToken;
-        }
+        return PREFIX_TOKEN;
     }
 
     @ManagedAttribute(description = "Suffix token")
     public String getSuffixToken() {
```

Comment:
getPrefixToken and getSuffixToken is removed from from SPI interface, but kept in implementation. Is this intention? These two methods in impl are unused, just exposing JMX and this is IMHO not needed to be in JMX now, when it is constant.